{{ cookiecutter.project_slug }}/.env.example

```diff
@@ -0,0 +1,11 @@
+DATABASE_URL=postgres://
```

Comment:
Hm... I don't really understand why it's here and is not conflicting (?), the `.env.example` is already there on `develop` o_O